common/source/docs/common-radio-control-calibration.rst

```diff
@@ -1,28 +1,85 @@
 .. _common-radio-control-calibration:
 
-============================================
-Radio Control Calibration in Mission Planner
-============================================
+=========================
+Radio Control Calibration
+=========================
 
-This article shows how to perform radio control calibration using
-*Mission Planner*.
+This article shows how to perform radio control calibration using Mission Planner
 
 Overview
 ========
 
-RC transmitters are used to control vehicle movement and orientation.
-Copter and Plane minimally control throttle, pitch, roll and yaw, while
-on Rover we just control throttle and roll. Each of these control
-signals are mapped to transmitter stick/switch(s) and in turn to
-autopilot channels from the connected receiver.
+RC transmitters allow the pilot to set the flight mode, control the vehicle's movement and orientation and also turn on/off auxiliary functions (i.e. raising and lowering landing gear, etc).
 
-Calibrating each of the transmitter controls/channels is a
-straightforward process - simply move each of the enabled
-sticks/switches through their full range and record the maximum and
-minimum positions.
+RC Calibration involves capturing each RC input channel's minimum, maximum and "trim" values so that ArduPilot can correctly interpret the input.
 
-Transmitter configuration
--------------------------
+Check the Transmitter's Setup
+-----------------------------
+
+- Ensure the battery is disconnected (this is important because it is possible to accidentally arm the vehicle during the RC calibration process)
+- Ensure the RC receiver is connected to the flight controller
+- Turn on your RC transmitter and if it has "trim tabs" ensure they are in the middle
```

Comment:
shouldn't the trim be middle before turn on ?

Reply:
Most tx now have digital trim...you can't be sure they are centered until you turn on tx